frontend/packages/console-shared/src/components/formik-fields/field-types.ts

```diff
@@ -97,6 +97,7 @@ export interface MultiColumnFieldProps extends FieldProps {
   emptyMessage?: string;
   headers: ({ name: string; required: boolean } | string)[];
   complexFields?: boolean[];
+  isMultiline?: boolean;
```

Comment:
As discussed on slack, but for posterity, this is not a valid solution. Almost all our usages of MultiColumnField are single lines suffering from this height issue. This prop would effectively always need to be on for every usage. 

As noted in the comments (and in Slack DMs) aim to see about removing the offending css and see if everything works fine. It's unclear why it was added but it seems to be an issue now (possible PF update of spacing to cause this issue)

Reply:
@andrewballantyne I updated and removed the undesired CSS block. I tested many areas like Pipelines, PipelineBuilder, Project Access which makes use of the Component and they all seem to work fine